client/src/components/App.jsx

```diff
@@ -9,10 +13,21 @@ class App extends React.Component {
     this.state = {};
     this.state.username = '';
     this.state.videos = [];
-  }
+    this.state.currentCategory = 'Featured';
+
+    this.handleOnRightClick = this.handleOnRightClick.bind(this);
+    this.handleOnLeftClick = this.handleOnLeftClick.bind(this);
 
+    this.handleOnChange = this.handleOnChange.bind(this);
+  }
 
   componentDidMount() {
+    document.querySelector('.custom-select-wrapper')
+      .addEventListener('click', function () {
+        this.querySelector('.custom-select').classList.toggle('open');
+        this.querySelector('.custom-select__trigger').classList.toggle('open');
```

Comment:
@Kraig @Nguyephi, I'm sure lines 25-28 get the job done, but I'm curious if there's a more React-way to write them. I usually bind event listeners to a div, but these lines are toggling classes and I think React can't work with multiple classes in a div without extra modules installed.

Reply:
I think in React it would be more common to have something like:
<div className={`"custom-select ${toggleClassName}"`}>
and toggleClassName would be '' or 'open'
(I'm not sure about the "'s in that div line...)

Reply:
Yes, I am working on refactoring my code to use state to alternate between css classes. Thank you @azdoolittle and @Kraig5.

Reply:
Wanted to chime in on this. In react theres the onClick that are great for event listener. onClick can be used on any tag (last I checked). ex: <button onClick={(e) => handleClick(e)}>Button</button>. Passing e (short for event) is the event your click will listen to and you can use it how you need to pull info from the dom similar to how you would with a query selector. Also define the handleClick function to do what you need and I believe you can add javascript to toggle. Not sure of a react way with css classes. Maybe some tinkering with styled component (because its inline) and you can maybe pass variables (based on the toggle) into the styling.